R/utils.R

```diff
@@ -113,7 +113,7 @@ ignore_files <- function(rfiles, path) {
   rfiles_relative <- sub("^[/]*", "", rfiles_relative)
 
   # Remove any files that match any perl-compatible regexp
-  patterns <- readLines(rbuildignore, warn = FALSE)
+  patterns <- scan(rbuildignore, what = "", sep = "\n", blank.lines.skip = TRUE, quiet = TRUE)
```

Comment:
Rather than switching to `scan()`, which is extremely complex, could you please just filter `""` out of patterns?